doc/source/appendices/tutorial.rst

```diff
@@ -23,199 +23,133 @@ Notes:
+* Set up a longer Run.  Modify ``npt`` in icepack_in.  ``npt`` defines the number of timesteps to run.  Details about namelist options are in the documentation (:ref:`case_settings`).
 
+* Modify a physics option.  Change the thermodynamics option from ktherm=2 to ktherm=1 in **icepack_in**, and set sw_redist=.true.  The intent here is to change the namelist option for the current experiment in the case directory.  What is different compared to your first run?  What happens if sw_redist = .false. with ktherm = 1?  Why?
 
-Change a Parameter in the Fortran Code
------------------------------------------
+* Undo your latest **icepack_in** changes
 
-Set up another case::
+* Change a Parameter in the Fortran Code.  Edit **icepack_mechred.F90** and set
 
-  ./icepack.setup --case ~/icepack-dirs/cases/icepack_test3 --mach <machine> --env <myenv>
+    ``fsnowrdg = c1    , & ! snow fraction that survives in ridging``.  
 
-Change to the source code directory::
+  Rebuild the code before running.  What is different about this run?  What do you think the fsnowrdg parameter is doing here?
 
-  cd columnphysics
+* Revert your latest code changes::
 
-Edit icepack_mechred.F90 to change the line
+    cd ~/Icepack
+    git status
+    git checkout columnphysics/icepack_mechred.F90
+    git status
 
-  fsnowrdg = p5    , & ! snow fraction that survives in ridging
+.. _tutorialfluff:
 
-to
+Add a New Tracer
+--------------------------------------
 
-  fsnowrdg = c1    , & ! snow fraction that survives in ridging
+In this exercise, add a new tracer associated with fluffballs.
+Call the tracer fluff and make it depend on ice area.  Follow the step-by-step instructions in the :ref:`addtrcr` documentation.
+Once you have implemented the model changes, be sure to add fluffballs output to the standard output diagnostics and turn on the
+fluff tracer.  Then update the timeseries plotting script to plot the fluffballs values over time.
 
-Build the code and submit the job.
+* First, set the initial value, physics, sources, and sinks of fluff to zero and make sure fluff values remain zero throughout the run
 
-* What is different about this run?
-* What do you think the fsnowrdg parameter is doing here?
+* Add some constant atmospheric forcing and review results
 
-Revert your code changes::
+* Change the dependency to ice volume, how do the results change?
 
-  cd ~/Icepack
-  git status
-  git checkout columnphysics/icepack_mechred.F90
-  git status
+* Modify the physics to create some physics processes, see isotopes or aerosols for some ideas
 
+**NOTE:** The file, **doc/source/tutorial/fluff.diff** in the Icepack repository, demonstrates code differences for this fluffball activity as implemented in a version of Icepack from July, 2024.  These code difference may not be directly applicable to other code versions, but these diffs provide an example of the typical code differences required to add the tracer, fluff.
```

Comment:
very minor corrections:
"These code differences..."
"example of the typical code modifications required..."